static/app/views/performance/newTraceDetails/traceModels/traceTree.tsx

```diff
@@ -1833,6 +1888,26 @@ export class TraceTreeNode<T extends TraceTree.NodeValue = TraceTree.NodeValue>
     return clone;
   }
 
+  filter(
+    node: TraceTreeNode<TraceTree.NodeValue>,
```

Comment:
Just a thought: Would it make sense to pass the node itself (or rather its value) through the predicate as well?

Reply:
I dont know, right now I dont think there is a need, but it should be a simple change if we need it